README.md

```diff
@@ -22,42 +23,39 @@ You can now start the admin, pupil-spa & pupil-api applications individually.
 
 ## Docker Compose
 
-There are 3 docker files...
+There are 2 docker-compose files...
 
 ### docker-compose.yml
-The default, which runs just SQL Server (main data store) and MongoDB (express session store)
+The default, which runs just SQL Server (main data store) and Redis (express session store)
 Stand up the database containers with: `docker-compose up`
 
-### docker-compose.admin-test.yml and docker-compose.pupil-test.yml
-Runs SQL Server, MongoDB, all web applications, and the pupil and admin test suites as separate containers.
-This compose file depends on `docker-compose.yml` and should not be run directly.
+### docker-compose-other-dev.yml
```

Comment:
could we call this `docker-compose-full.yml` please, feels more specific

Reply:
I would argue that it is not "full" as this compose file lacks Redis and mssql.  To bring everything up requires using 2 compose files - now that is "full".

How about calling it "docker-compose-apps.yml" as one file is for data stores and one for apps.

Reply:
good point, sounds accurate to me 👍

Reply:
Changed